specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,268 @@
+    COREWEBVIEW2_BROWSING_DATA_KINDS_PASSWORD_AUTOSAVE |
+    COREWEBVIEW2_BROWSING_DATA_KINDS_BROWSING_HISTORY | 
+    COREWEBVIEW2_BROWSING_DATA_KINDS_SETTINGS,
+} COREWEBVIEW2_BROWSING_DATA_KINDS;
+
+[uuid(DAF8B1F9-276D-410C-B481-58CBADF85C9C), object, pointer_default(unique)]
+interface ICoreWebView2Profile : IUnknown {
+
+  /// Clear browsing data based on a data type. This method takes two parameters, 
+  /// the first being a mask of one or more `COREWEBVIEW2_BROWSING_DATA_KINDS`. OR operation(s) 
+  /// can be applied to multiple `COREWEBVIEW2_BROWSING_DATA_KINDS` to create a mask 
+  /// representing those data types. The browsing data kinds that are supported 
+  /// are listed below. These data kinds follow a hierarchical structure in which 
+  /// nested bullet points are included in their parent bullet point's data kind.
+  /// Ex: DOM storage is encompassed in site data which is encompassed in the profile data. 
+  /// * Profile
+  ///   * Site Data
+  ///     * DOM Storage: App Cache, File Systems, Indexed DB, Local Storage, Web SQL, Cache 
+  ///         Storage
+  ///     * Cookies 
+  ///   * HTTP Cache 
+  ///   * Download History
+  ///   * General Autofill 
+  ///   * Password Autosave
+  ///   * Browsing History
+  ///   * Settings  
+  /// The completed handler will be invoked when the browsing data has been cleared and will 
+  /// indicate if the specified data was properly cleared.
+  /// If the WebView object is closed before the clear browsing data operation 
+  /// has completed, the handler will be released, but not invoked. In this case 
+  /// the clear browsing data operation may or may not be completed. 
+  /// ClearBrowsingData clears the `dataKinds` for all time. 
+  HRESULT ClearBrowsingData(
+      [in] COREWEBVIEW2_BROWSING_DATA_KINDS dataKinds,
+      [in] ICoreWebView2ClearBrowsingDataCompletedHandler* handler);
+  
+  /// ClearBrowsingDataInTimeRange behaves like ClearBrowsingData except that it
+  /// takes in two additional parameters for the start and end time for which it 
+  /// should clear the data between.  The `startTime` and `endTime` 
+  /// parameters correspond to the number of seconds since the UNIX epoch. 
+  ///
+  /// \snippet AppWindow.cpp ClearBrowsingData
+  HRESULT ClearBrowsingDataInTimeRange(
+      [in] COREWEBVIEW2_BROWSING_DATA_KINDS dataKinds, 
+      [in] double startTime,
+      [in] double endTime, 
+      [in] ICoreWebView2ClearBrowsingDataCompletedHandler* handler);
+}
+
+/// The caller implements this interface to receive the `ClearBrowsingData` result.
+[uuid(27676699-FE17-4E2B-8C1B-267395A04ED5), object, pointer_default(unique)]
+interface ICoreWebView2ClearBrowsingDataCompletedHandler : IUnknown {
+  /// Provide the completion status of the corresponding asynchronous method. 
+  HRESULT Invoke([in] HRESULT errorCode);
+}
+
+```
+### .NET, WinRT
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+    [Flags] enum CoreWebView2BrowsingDataKinds
```

Comment:
Why no DomStorage and Profile groupings?

Reply:
There is a DomStorage value below and a Profile value below. Do you mean something else? Maybe its confusing looking because this is generated and the values are precomputed so you don't see the A = B | C | D | E